packages/frontend/src/features/sqlRunner/store/index.ts

```diff
@@ -1,13 +1,14 @@
 // FIXES ts2742 issue with configureStore
 // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
-import type * as rtk from '@reduxjs/toolkit';
```

Comment:
we need to add this back so the build passes pls 👍

Reply:
WFT, merges today